packages/web3-shared/evm/src/utils/address.ts

```diff
@@ -86,3 +92,8 @@ export function isENSContractAddress(contract_address: string) {
     const { ENS_CONTRACT_ADDRESS } = getENSConstants()
     return isSameAddress(contract_address, ENS_CONTRACT_ADDRESS)
 }
+
+export function checkInWhiteliss(chainId = ChainId.Mainnet, address: string) {
```

Comment:
Move to `packages/web3-providers/src/GoPlusLabs/index.ts` or packages/web3-providers/src/GoPlusLabs/helpers.ts

Reply:
This method is not shared with other parts. We don't have to put it here.